docs/docs/examples/bamboo/REMOTE_AGENTS.md

```diff
@@ -8,23 +8,64 @@ You can learn more details about remote agents on the [official documentation pa
 
 !!!warning "Bamboo server prerequisites"
 
-    * Your Bamboo server instance must be a fully licensed Bamboo Data Center instance
-    * The bamboo server instance must be fully configured
+    * The Bamboo server instance must use a valid Bamboo Data Center instance license and be fully configured
     * The Bamboo server instance must have `security token verification` **enabled**
     * The Bamboo server instance must have `remote agent authentication` **disabled**
 
-## Installation
+## Deployment
 
 Steps required for deploying a remote agent
 
-### Overview
-
 1. Configure Bamboo server for remote agent support
 2. Deploy agent
 
-### Steps
+### 1. Configure Bamboo server
+
+There are 2 approaches for doing this:
+
+* Automatically when deploying Bamboo server
+* Manually via Bamboo server agent settings
+
+#### Automatically
+
+When initially deploying Bamboo server its `values.yaml` can be configured to:
+
+* disable `remote agent authentication`
+* define a custom `security token` 
+
+This will allow remote agents that are configured with the same security token to automatically join the cluster. 
+
+First, create a secret to store a custom security token with which remote agent(s) should authenticate to the Bamboo server. 
+
+!!!info "Security token format"
+    
+    The security token should be set to a 40-character hexadecimal string. The following command can be used to do this:
+    ```
+    xxd -l 20 -p /dev/urandom
```

Comment:
@tarka @nanux thoughts on this approach for generating a 40 char hex string?

Reply:
Seems fine.